packages/lesswrong/components/ea-forum/onboarding/EAOnboardingInput.tsx

```diff
@@ -24,34 +25,42 @@ export const EAOnboardingInput = ({
   value,
   setValue,
   placeholder,
+  As="input",
```

Comment:
We just reuse the onboarding input for the emoji popup as the styles are mostly the same. Probably we should move this input into a shared directory at some point, but that can be a different PR.